src/views/auth/LoginView.vue

```diff
@@ -81,38 +103,14 @@ if (redirect !== null) {
   logger.info('Redirect after login: ' + redirect)
 }
 
-// if (token !== null) {
-//   onMounted(async () => {
-//     other_login.value = true
-//     let rs
-//     try {
-//       rs = await api.v2.auth.oauth.login.token(token)
-//     } catch (e) {
-//       sendErrorMessage('登录失败: ' + e)
-//       router.push('/auth/login')
-//     }
-//     if (!rs) return
-//     if (rs.status === 200) {
-//       message.success(rs.data.username + '，欢迎回来！')
-//       userData.commit('set_token', rs.data.token)
-//       userData.commit('set_user_info', rs.data)
-//       router.push(redirect || '/dashboard')
-//     }
-//   })
-// }
-
-// LoCyan OAuth 2.0
-// There is no need
-// function oauthLogin() {
-//   oauthLogin_loading.value = true
-//   window.location.href =
-//     'https://api-v2.locyanfrp.cn/api/v2/oauth/authorize?redirectUrl=http://' +
-//     window.location.host +
-//     '/auth/login'
-// }
+watch(token, (oldToken, newToken) => {
```

Comment:
Maybe using `watch` hook is not a good idea.

Reply:
https://github.com/Astrian/cfturnstile-vue3 offered a callback event, which allows us avoid the use of `watch` hook.

Reply:
I've already tested this package, it can't reset the captcha widget after the n-model re-rendered, will case unexpectedly behavior.